azure-storage-blob/azure/storage/blob/blockblobservice.py

```diff
@@ -574,6 +574,20 @@ def create_blob_from_stream(
                 progress_callback(0, count)
 
             data = stream.read(count)
+            data_chunk = data  # to store the chunk of data read from stream each time
+
+            # keep reading from stream util length of data >= count or reaching the end of stream
+            while len(data) < count and len(data_chunk) is not 0:
+                data_chunk = stream.read(count)
+                data += data_chunk
+
+            if len(data) < count:
+                raise ValueError('Parameter:count you provided is more than the actually data exist in stream,'
```

Comment:
This sentence reads a bit awkward (use of "you" feels a bit informal for error messages and "actually data exist in stream" is not quite correct). Perhaps 'Parameter:count is greater than the amount of data in the stream."

Reply:
That makes more sense! Absolutely it is more professional to describe in the way you proposed